### ISSUE TITLE
Prepare gnverifier to v1.0.0 release

### ISSUE THREAD

**Issue by dimus** [gnames/gnverifier#96]
gnapi also needs to be upedated

**Comment by abubelinha** [comment]
Hi @dimus 
Not sure if this is related but my scripts based on this url are failing:
https://verifier.globalnames.org/api/v0/verifications/

As mentioned in #68, [https://github.com/gnames/gnverifier/issues/82#issuecomment-1035404398](#82) and #90, I understood that was the suggested api access to rely on.

Have you made any changes that we need to be aware of?
(I hope this only means we just need to change v0 to v1 in our urls)

Thanks!

**Comment by dimus** [comment]
@abubelinha yes, it is related, you can change v0 to v1 and it should work again

**Comment by dimus** [comment]
I am thinking if I should make a redirect of /api/v0 to /api/v1, until there is again a developing version in place, however it would just postpone this problem to some random time.

**Comment by abubelinha** [comment]
Thanks for your confirmation.
I updated the url and made a simple test: my scripts seem to be working again.

But just in case I am missing something advanced options, I prefer to ask: 
Are there any request/response API parameters that might have been modified during the last weeks?

Thanks a lot in advance

**Comment by dimus** [comment]
@abubelinha I was trying to stabilize API for a few months, and your feedback played an important role in that process. Moving to v1.0.0 would indicate that API is considered stable enouth, that parameters will stop changing, only new parameters will be added. So movement to v1.0.0 only included changing urls and version numbers, other than that it is the same as it was for v1.0.0-RC1.

Of course nothing prevents me to find out that I made some huge mistake and API must become backward incompatible. However in such case such change should really be important, and this time v1 will continue to be served. In addition v2.0.0 will be also served in parallel.

Sorry for the troubles of this transition, but I think it was important to promise stability. For example gnparser reached v1.0.0 (a stable version) one year and seven months ago and its API and schema were stable since then.
